airbyte-cdk/python/airbyte_cdk/sources/utils/transform.py

```diff
@@ -2,14 +2,14 @@
 # Copyright (c) 2021 Airbyte, Inc., all rights reserved.
 #
 
+import logging
 from distutils.util import strtobool
 from enum import Flag, auto
 from typing import Any, Callable, Dict, Mapping, Optional
 
-from airbyte_cdk.logger import AirbyteLogger
 from jsonschema import Draft7Validator, validators
 
-logger = AirbyteLogger()
+logger = logging.getLogger("airbyte")
```

Comment:
A few places were still using the old, deprecated `AirbyteLogger` - aside from deprecation warnings, this was causing some circular dependency issues when I refactored the `AirbyteSecretHelper` out from `logger.py` into `airbyte_secrets_utils.py`.

Reply:
clarified on slack, I misunderstood, we're all good here
i.e. 
> ~~ensure~~ we aren't breaking AirbyteLogger with this change